test/minitest/xmfun/util/url_parser_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'mini_test_helper'
+
+module Xmfun
+  module Util
+    class TestUrlParser < Minitest::Test
+      def setup
+        @urlParser = Xmfun::Util::UrlParser
+      end
+
+      def test_parse__return_url_for_corresponding_type
```

Comment:
Assignment Branch Condition size for test_parse__return_url_for_corresponding_type is too high. [15.62/15]